modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/CarouselView.tsx

```diff
@@ -3,14 +3,99 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import React from 'react';
+import React, {useState} from 'react';
+import {Card} from '../../../../../../../frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards';
```

Comment:
This is not the right way to import. It should be `import {Card} from '@liferay/frontend-data-set-web';`, along with export from frontend-data-set-web module.

Reply:
That will be my next question, thank you!